noir_stdlib/src/hash/sha256.nr

```diff
@@ -89,31 +89,42 @@ pub fn sha256_var<let N: u32>(msg: [u8; N], message_size: u64) -> [u8; 32] {
     let mut h: [u32; 8] = [1779033703, 3144134277, 1013904242, 2773480762, 1359893119, 2600822924, 528734635, 1541459225]; // Intermediate hash, starting with the canonical initial value
     let mut msg_byte_ptr = 0; // Pointer into msg_block
 
-    if num_blocks == 0 {
```

Comment:
This if statement before the loop has been removed in favor of a `if modulo != 0` after the loop to account for both when N < BLOCK_SIZE and when N > BLOCK_SIZE @guipublic